addons/account/models/account_report.py

```diff
@@ -558,6 +558,14 @@ class AccountReportExpression(models.Model):
         ),
     ]
 
+    @api.constrains('carryover_target')
+    def _check_carryover_target(self):
+        for expression in self:
+            if expression.carryover_target and not expression.label.startswith('_carryover_'):
+                raise UserError("You cannot use the field carryover_target in an expression that does not have the label starting with _carryover_")
```

Comment:
Don't forget to call the translation function ;) (below as well)

You'll then also need to update the .pot file

Reply:
Does it really need to be translated because it shouldn't be visible in stable

Reply:
It's true those exceptions won't be raised in common flows, but for partners doing more advanced configuration by hand, it could happen ;)